cap-ci/cap-nightly-caasp.yaml

```diff
@@ -0,0 +1,67 @@
+workertags:
+  caasp4: suse-internal
+
+jobs:
+  deploy-k8s: false
+  deploy-kubecf: true
+  smoke-tests: true
+  cf-acceptance-tests-brain: true
+  sync-integration-tests: true
+  minibroker-integration-tests: true
+  cf-acceptance-tests: true
+  upgrade-kubecf: true
+  deploy-stratos: false
```

Comment:
Why not Stratos?

Reply:
Stratos was removed from CI definition: https://github.com/SUSE/cap/blob/master/cap-ci/cap-pre-release-caasp.yaml#L13
As I said, nightly yamls are just a replica of pre-release CI, so any changes to the CI definition will have to be done in separate card. 
This card just focuses on master build regex and schedule.
Stratos is suppose to be tested by its own CI https://confluence.suse.com/pages/viewpage.action?spaceKey=CAP&title=CAP+CI+Overview